### PR TITLE
feat: Make securityContext and podSecurityContext configurable

### PR DESCRIPTION
**Pull request by legal90**
#### What this PR does / why we need it:
This PR makes it possible to customize `securityContext` on both container and pod level:
- https://kubernetes.io/docs/tasks/configure-pod-container/security-context/#set-the-security-context-for-a-container
- https://kubernetes.io/docs/tasks/configure-pod-container/security-context/#set-the-security-context-for-a-pod

Added 2 values with their defaults, keeping the backward compatibility:
- `securityContext`
- `podSecurityContext`

#### Special notes for your reviewer:
No breaking changes, could be merged safely.

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] Chart Version bumped
- [x] Variables are documented in the README.md

### CODE REVIEW

**Comment by legal90** [comment]
Hi @ekampf,  
Could you please review this PR?

**Comment by ekampf** [comment]
@legal90 can you please fix the chart version to `0.1.29` ?

**Comment by ekampf** [comment]
nm, did it for you :)

File: stable/connector/values.yaml [hunk 1]
```diff
@@ -75,6 +75,25 @@ env: {}
...
 
+## Security context for the Connector container
+## https://kubernetes.io/docs/tasks/configure-pod-container/security-context/#set-the-security-context-for-a-container
+##
+securityContext:
+  allowPrivilegeEscalation: false
+  capabilities:
+    drop: ["ALL"]
+  runAsNonRoot: true
+  runAsUser: 65532
```
[the remark this answers is not in the document]
Comment (legal90) [reply stable/connector/values.yaml:86]: Sure! Done 👍

**Review by twingate-blee — APPROVED** [review]
Looks good to me. Thanks

**Pull request merged**